ibis-server/app/routers/v2/ibis.py

```diff
@@ -27,12 +27,15 @@ def query(
     data_source: DataSource,
     dto: QueryDTO,
     dry_run: Annotated[bool, Query(alias="dryRun")] = False,
+    limit: int = 100,
```

Comment:
I think the default value could be `unlimit`. It can also be used by the user who want to get all result.